anker/src/metrics.rs

```diff
@@ -0,0 +1,39 @@
+use borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
+use lido::token::StLamports;
+use serde::Serialize;
+
+use crate::token::{self, MicroUst};
+
+#[repr(C)]
+#[derive(
+    Clone, Debug, Default, BorshDeserialize, BorshSerialize, BorshSchema, Eq, PartialEq, Serialize,
+)]
+pub struct Metrics {
+    /// Total swapped amount of StSOL to UST.
+    #[serde(rename = "swaped_rewards_total_st_lamports")]
+    pub swapped_rewards_st_sol_total: StLamports,
+
+    /// Total swapped amount of UST to StSOL.
+    #[serde(rename = "swapped_rewards_ust_total_microust")]
+    pub swapped_rewards_ust_total: MicroUst,
+}
```

Comment:
:+1: Great idea to have stSOL and UST totals, if we sample that over time then we can compute the average exchange rate we got over any time window we want, neat.